controller/DeliveryRunner.php

```diff
@@ -162,18 +161,7 @@ public function finishDeliveryExecution()
                 $stateService->finish($deliveryExecution);
             }
         }
-        $redirectUrl = LTIDeliveryTool::singleton()->getFinishUrl($this->getLtiMessage($deliveryExecution), $deliveryExecution);
+        $redirectUrl = LTIDeliveryTool::singleton()->getFinishUrl($deliveryExecution);
```

Comment:
At the moment it's not possible, LTIDeliveryTool doesn't extend Configurable service. 
I want to refactor this part as well but didn't want to include it into this PR to do not make it too big.

Reply:
You already did it :) and you've increased major version

Reply:
This class is used in ~15 other classes. It will require quite bigger refactoring which is not directly related to this PR. It won't be major  version increase because we will  not change any of existing interfaces, singleton method will be preserved (and deprecated) for backward compatibility. 
I  think it will be easier to review 2 smaller PRs then 1 with 25+ files.